scripts/upgrade_rstudio_pods.sh

```diff
@@ -1,27 +1,17 @@
 #!/usr/bin/env bash
 set -ex
 
-if [ $# -lt 1 ]; then
-  echo 1>&2 "$0: Arguments: DOCKER_TAG"
-  exit 2
-fi
-
-DOCKER_TAG=$1
+DOCKER_TAG=${1:-"latest"}
```

Comment:
What is [this sorcery](https://stackoverflow.com/questions/27445455/what-does-the-colon-dash-mean-in-bash) - nice 👍